### DISCUSSION TITLE
Bad quality audio for flac

### DISCUSSION THREAD

**Discussion by cnic023** [ampache/ampache#3960] [Q&A] [answered]
I'm running Ampache 6.2.0 and all my files are *.flac. What may be relevant configs are:
transcode_flac allowed (I don't want transcode with flac).
transcode	default
allow transcoding default
rate limit 8192
transcode bitrate 32
playback type web player

PROBLEM is that the audio quality for higher frequencies e.g. violin is very bad, very tinny. Also, it often sounds like being underwater. There is a song by the late, great Leon Russell called Manhattan Island Serenade. At the beginning there is the sound of rain falling but it really sounds like little pieces of metal dropped from a great height!
When I play the same music off a standard MS Windows player using the files copied over to my laptop from the server, the quality is superb, so I guess the problem is not in the rip. Trouble is I don't understand/know what's going on under the hood. I would expect the music data to get to my laptop a bit slower (being flac as opposed, say, to mp3, because more data) but I didn't expect diminished quality as I assumed some sort of buffer was at work. I also cast (bluetooth) the output to a Receiver (we used to call them amplifiers in the olden days) and its the same problem. When I do the same thing with the standard MS player from my laptop (as described above), its great listening, no problems.
Any ideas? I hope it's a configuration problem!

**Comment by mitchray** [comment, marked as answer]
Going by what you shared, change ```transcode_flac``` to 'false' or comment that line out to stream the raw flac, it is currently being transcoded to 32kbps which explains the underwater sounding result

Alternatively bump that up to 128+

**Comment by cnic023** [comment]
Hey, it worked! I thought the options were only ‘required’ or ‘allowed’. Many, many thanks.
